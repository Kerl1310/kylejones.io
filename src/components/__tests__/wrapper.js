import React from 'react'
import renderer from 'react-test-renderer'

import Wrapper from '../wrapper'

describe('Wrapper', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Wrapper className="my-wrapper" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
