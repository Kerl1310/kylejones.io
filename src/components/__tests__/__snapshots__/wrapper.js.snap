// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper renders correctly 1`] = `
<main
  className="sc-bdVaJa hNWmWv my-wrapper"
  role="main"
/>
`;
